### ISSUE TITLE
Remove duplicate calls on Explore

### ISSUE THREAD

**Issue by techsmyth** [alkem-io/client-web#2887]
## Description

As a user I want a fast page load

## Acceptance criteria

- [ ] Card Template is only retrieved when the user goes to add a new card. And only once per Callout. 
- [ ] Canvases list only retrieved when the user goes to add a new canvas

## Additional Context
The following page as one Card Callout but 3 times the data is retrieved: https://dev.alkem.io/neil2/explore

![image.png](https://images.zenhubusercontent.com/5ecb9817aeb589610134bd43/8b405998-ae29-4703-a14e-a06f69d658fe)

**Comment by techsmyth** [comment]
Closing as is a duplicate of 2902
